Tests/Fixtures/TestProject/Project.xcodeproj/project.pbxproj

```diff
@@ -1652,8 +1652,8 @@
 			isa = PBXNativeTarget;
 			buildConfigurationList = 77CE5B5E5DEAC820254D484C /* Build configuration list for PBXNativeTarget "App_macOS" */;
 			buildPhases = (
-				96BB43F4706B031DA45166E8 /* Sources */,
```

Comment:
This change is caused by the added property `resourcesBeforeSourcesBuildPhase` in `[Tests/Fixtures/TestProject/project.yml](https://github.com/yonaskolb/XcodeGen/pull/1351/files#diff-210015ff17dd32e8499b1f864d8c6c2de0ff90b5dbcee46601a84528fe59de7f)